README.md

```diff
@@ -21,24 +21,63 @@ Refer to the individual manpages for details.
 
 ## Installation
 
-The RTP tools should compile on any POSIX-compliant
-platform supporting sockets, as well as on Windows.
+RTP tools should compile and run on any POSIX compatible system,
+as well as on Windows. It is also possible that your OS already
+provides a prebuilt package of RTP tools.
 
-### UNIX
+### configure
 
-If building from git, run the following first to regenerate the build tools:
+Run `./configure` to configure the build for your system.
+This will produce three files:
 
-```
-aclocal
-automake --add-missing --force-missing
-autoconf
-```
+* `config.h` containing the `#include` and `HAVE_` lines
+* `config.log` containing the details of autodetection
+* `Makefile.local` which defines `CC`, `PREFIX` and the like
 
-- Run `./configure` to configure rtptools for your system.
-- Run `make` to build rtptools.
-- Run `make install` to install.
-- Run `make uninstall` to uninstall.
-- Run `make rpm` to create a rpm package in `./rpmbuild`
+Read the standard output and `Makefile.local`.
+If these look different from what you expected,
+read `configure.local.example`, create `configure.local`,
+and run `./configure` again.
+
+The build system does not use GNU autoconf. All modern operating systems
+are now reasonably close to POSIX and do not need arcane shell magic
+any longer. If your system does need such magic, consider upgrading
+to reasonably modern POSIX-compliant tools rather than asking for
+autoconf-style workarounds.
```

Comment:
You don't like autotools and that's fine. But is this text really necessary? README.md is one of the first files a new user encounters.

Reply:
I have no objections to removing this. I felt it appropriate to inform the user
that the `./configure` script is not what he might be expecting from elsewhere.
I will remove it.